__tests__/utils/CoreServicesUtils.spec.ts

```diff
@@ -1,48 +1,53 @@
 
-import { CoreServicesGeoNamesMapping, coreServicesOrgUrlPrefix, getCoreServicesGeoName, isCoreServicesOrgUrl, nonProductionDynamicsLocationCode, unqOrgUrlPattern } from "../../src/utils/CoreServicesUtils";
+import { CoreServicesGeoNamesMapping, coreServicesOrgUrlPrefix, createCoreServicesOrgUrl, getCoreServicesGeoName, isCoreServicesOrgUrl, nonProductionDynamicsLocationCode, unqOrgUrlPattern } from "../../src/utils/CoreServicesUtils";
 
 describe("CoreServicesUtils", () => {
     it("unqOrgUrlPattern should be able to retrieve the location code from the an UNQ OrgUrl", () => {
         const locationCode = "crm";
-        const orgUrl = `https://unq[orgId]-${locationCode}.omnichannelengagementhub.com`;
+        const orgUrl = `https://unq1234-${locationCode}.omnichannelengagementhub.com`;
         const result = unqOrgUrlPattern.exec(orgUrl);
 
-        expect(result).toBeDefined();
+        expect(result).not.toBe(null);
         if (result) {
             expect(result[1]).toBe(locationCode);
+            expect(createCoreServicesOrgUrl("1234",getCoreServicesGeoName(locationCode))).toBe(`https://m-1234.us.omnichannelengagementhub.com`)
         }
     });
 
     it("unqOrgUrlPattern should be able to retrieve the location code from the an custom UNQ OrgUrl", () => {
         const locationCode = "crmtest";
-        const orgUrl = `https://[custom]-${locationCode}.omnichannelengagementhub.com`;
+        const orgUrl = `https://unq1234-${locationCode}.oc.crmlivetie.com`;
         const result = unqOrgUrlPattern.exec(orgUrl);
         
-        expect(result).toBeDefined();
+        expect(result).not.toBe(null);
         if (result) {
             expect(result[1]).toBe(locationCode);
+            expect(createCoreServicesOrgUrl("1234",getCoreServicesGeoName(locationCode))).toBe(`https://m-1234.test.omnichannelengagementhub.com`)

```

Comment:
Should this be extracted to a new test?

Reply:
there is already another test to validate 'getCoreServicesGeoName'